pyrepl/keymaps.py

```diff
@@ -77,8 +77,7 @@
         (r"\<home>", "home"),
         (r"\<f1>", "help"),
         (r"\EOF", "end"),  # the entries in the terminfo database for xterms
-        (r"\EOH", "home"),  # seem to be wrong.  this is a less than ideal
-        # workaround
+        (r"\EOH", "home"),  # seem to be wrong. this is a less than ideal workaround
```

Comment:
Now it appears with "home" only, but likely is also meant for "end".  (just from what I think/see, no idea really).
I wanted to ensure that the comment's meaning/location/reference is kept (which was given before due to whitespace alignment), but now it is even less clear.

Reply:
Ya I agree.  It looks like the comment spans multiple lines and starts with "the entries in the terminfo database.."  IMO it should be on a separate line, before the end & home lines.

Reply:
https://github.com/pypy/pyrepl/pull/48/commits/0262f36a1fa1815a9789706fac50190ca58eef75 fix that

should there be new line between `help` and comment? if yes, i will create another commit for that